projects/ad35xxr_evb/zed/README.md

```diff
@@ -1,10 +1,12 @@
-# AD35XXR-EVB/Zed HDL Project
+<!-- no_build_example, no_dts, no_no_os -->
+
+# AD35XXR-EVB/ZED HDL Project
+
+- VADJ with which it was tested in hardware: 1.8V
 
 ## Building the project
 
 ```
 cd projects/ad35xxr_evb/zed
 make
-```
-
-Corresponding Linux device tree: TO BE ADDED
+```
```

Comment:
`Corresponding Linux device tree: [zynq-zed-adv7511-ad3552r-hs.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-ad3552r-hs.dts)`

Add this, as it seems that the dts got published in the meantime.